PersonApi.Tests/V1/Helper/Constants.cs

```diff
@@ -8,18 +8,20 @@ public static class Constants
     {
         public static Guid ID { get; } = Guid.NewGuid();
         public const Title TITLE = Title.Mr;
+        public const Title PRETITLE = Title.Mr;
         public const string PREFIRSTNAME = "Bob";
```

Comment:
I know it's not in this PR, but it's probably named incorrectly too - shouldn't it be `PREFFIRSTNAME `?